packages/errors/src/context.ts

```diff
@@ -288,6 +290,13 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<
             path?: string;
             value: bigint;
         };
+        [SOLANA_ERROR__SIGNER_ADDRESS_CANNOT_HAVE_MULTIPLE_SIGNERS]: {
+            address: string;
+        };
+        [SOLANA_ERROR__SIGNER_EXPECTED_CONFORMING_SIGNER]: {
+            address: string;
+            interfaceName: string;
```

Comment:
Not super sure that this was the right thing to do, but you tell me @lorisleiva.